### PR TITLE
test(VBadge): cypress spec

### PR DESCRIPTION
**Pull request by ElijahKotyluk**
## Description

  - Added RTL support
  - Added Theme support

## Motivation and Context

Provide e2e testing for `VBadge`.

## How Has This Been Tested?

Cypress
## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Review by johnleider — APPROVED** [review]

File: packages/vuetify/src/components/VBadge/VBadge.tsx [hunk 1]
```diff
@@ -9,10 +9,13 @@ import { makeRoundedProps, useRounded } from '@/composables/rounded'
 import { makeTagProps } from '@/composables/tag'
 import { makeTransitionProps, MaybeTransition } from '@/composables/transition'
 import { useBackgroundColor, useTextColor } from '@/composables/color'
+import { useLocale } from '@/composables/locale'
 
 // Utilities
 import { computed, toRef } from 'vue'
 import { convertToUnit, defineComponent, pick } from '@/util'
+import { makeThemeProps, useTheme } from '@/composables/theme'
+import { useRtl } from '@/composables/rtl'
```
Comment (johnleider) [inline packages/vuetify/src/components/VBadge/VBadge.tsx:18]: move under // Composables

File: packages/vuetify/src/components/VBadge/VBadge.tsx [hunk 2]
```diff
@@ -56,13 +59,17 @@ export const VBadge = defineComponent({
...
     ...makeTagProps(),
+    ...makeThemeProps(),
     ...makeTransitionProps({ transition: 'scale-rotate-transition' }),
   },
 
   setup (props, ctx) {
     const { backgroundColorClasses, backgroundColorStyles } = useBackgroundColor(toRef(props, 'color'))
-    const { roundedClasses } = useRounded(props)
+    const { isRtl } = useRtl()
+    const { roundedClasses } = useRounded(props, 'v-badge')
```
Comment (johnleider) [inline packages/vuetify/src/components/VBadge/VBadge.tsx:69]:
```suggestion
    const { roundedClasses } = useRounded(props)
```

File: packages/vuetify/src/components/VBadge/VBadge.tsx [hunk 3]
```diff
@@ -56,13 +59,17 @@ export const VBadge = defineComponent({
...
+    const { t } = useLocale()
     const { textColorClasses, textColorStyles } = useTextColor(toRef(props, 'textColor'))
+    const { themeClasses } = useTheme()
```
Comment (johnleider) [inline packages/vuetify/src/components/VBadge/VBadge.tsx:72]:
```suggestion
    const { themeClasses } = useTheme(props)
```

**Review by johnleider — CHANGES_REQUESTED** [review]

File: packages/vuetify/src/components/VBadge/VBadge.tsx (hunk 3 above)
Comment (ElijahKotyluk) [reply packages/vuetify/src/components/VBadge/VBadge.tsx:72]: `useTheme()` no longer takes props as an argument.

**Review by johnleider — APPROVED** [review]

**Pull request merged**